samples/sampler/stories/default/Popup.js

```diff
@@ -4,32 +4,45 @@ import {boolean, select, text} from '@enact/storybook-utils/addons/knobs';
 import React from 'react';
 import {storiesOf} from '@storybook/react';
 
+import Button from '@enact/agate/Button';
 import {Popup, PopupBase} from '@enact/agate/Popup';
 
 const Config = mergeComponentMetadata('Popup', PopupBase);
 
 storiesOf('Agate', module)
 	.add(
 		'Popup',
-		() => (
-			<div>
-				<Popup
-					centered={boolean('centered', Config)}
-					closeButton={boolean('closeButton', Config)}
-					noAnimation={boolean('noAnimation', Config)}
-					noAutoDismiss={boolean('noAutoDismiss', Config)}
-					onClose={action('onClose')}
-					onHide={action('onHide')}
-					open={boolean('open', Config)}
-					scrimType={select('scrimType', ['none', 'translucent', 'transparent'], Config, 'translucent')}
-					spotlightRestrict={select('spotlightRestrict', ['self-first', 'self-only'], Config, 'self-only')}
-					title={text('title', Config, 'Title')}
-				>
-					<div>{text('children', Config, 'Hello Popup')}</div>
-				</Popup>
-				Use KNOBS to interact with Popup.
-			</div>
-		),
+		() => {
+			const buttons = boolean('buttons', Config, false);
+
+			return (
+				<div>
+					<Popup
+						centered={boolean('centered', Config)}
+						closeButton={boolean('closeButton', Config)}
+						noAnimation={boolean('noAnimation', Config)}
+						noAutoDismiss={boolean('noAutoDismiss', Config)}
+						onClose={action('onClose')}
+						onHide={action('onHide')}
+						open={boolean('open', Config)}
+						position={select('position', ['center', 'top'], Config)}
+						scrimType={select('scrimType', ['none', 'translucent', 'transparent'], Config, 'translucent')}
+						spotlightRestrict={select('spotlightRestrict', ['self-first', 'self-only'], Config, 'self-only')}
+						title={text('title', Config, 'Title')}
+					>
+						<div>{text('children', Config, 'Hello Popup')}</div>
+						{buttons ? (
+							<buttons>
+								<Button>NO</Button>
+								<Button>YES</Button>
+							</buttons>
```

Comment:
`buttons` is true or undefined. It's not a component name.

Anyway, I hope that we could have Popup qa sampler with buttons because it is not a basic usage of Popup.

Reply:
Hello @daniel-stoian-lgp 
I updated sampler and qa sample for Popup according to my comment above.
But the current look is different from the GUI. Could you update it?

Reply:
Yes, I will update. The buttons must be declared inside a <buttons> block in order to get the styling

Reply:
Oh I misunderstood. You used `buttons` as a slot not a tag name.